scripts/generate-app-config/generatePagesConfig.js

```diff
@@ -136,6 +136,104 @@ const buildPageConfig = (filePaths, generatePagesOptions, namespace) => (
   }, {})
 );
 
+const getScreenshotPatterns = generatePagesOptions => (
+  generatePagesOptions.searchPatterns.reduce((acc, { root }) => {
+    const rootPath = root.replace(/[\\]/g, '/');
+    acc.push({
+      pattern: `${rootPath}/packages/*/tests/wdio/__snapshots__/reference/**/*.png`,
+      // build out a regex for the entrypoint mask.
+      entryPoint: `${rootPath}/packages/[^/]*/tests/wdio/__snapshots__/reference/`,
+    });
+    acc.push({
+      pattern: `${rootPath}/tests/wdio/__snapshots__/reference/**/*.png`,
+      // build out a regex for the entrypoint mask.
+      entryPoint: `${rootPath}/tests/wdio/__snapshots__/reference/`,
+    });
+
+    return acc;
+  }, [])
+);
+
+const createEvidenceParent = packageNamespace => (
+  {
+    name: startCase(packageNamespace),
+    path: `/${kebabCase(packageNamespace)}`,
+    group: '',
+    pages: {},
+  }
+);
+
+const createLocaleParent = (packageNamespace, locale) => (
+  {
+    name: startCase(locale),
+    path: `/${kebabCase(packageNamespace)}/${kebabCase(locale)}`,
+    group: '',
+    pages: {},
+  }
+);
+
+const createEvidencePage = (packageNamespace, locale, name) => (
+  {
+    name: startCase(name),
+    path: `/${kebabCase(packageNamespace)}/${kebabCase(locale)}/${kebabCase(name)}`,
+    group: '',
+    content: {},
+    type: 'evidence',
+  }
+);
+
+const buildTestEvidenceConfig = (generatePagesOptions, namespace) => {
+  const patterns = getScreenshotPatterns(generatePagesOptions);
```

Comment:
Could we push these search patterns into the [finding patterns/globing that happens here](https://github.com/cerner/terra-dev-site/pull/109/files#diff-92bf9f0a216864327aecb81490a4575aR294). I feel like some of this duplicated

Reply:
Though I see the naming differs below...